lib/ansible/modules/file.py

```diff
@@ -63,9 +63,9 @@
   force:
     description:
     - >
-      Force the creation of the symlinks in two cases: the source file does
+      Force the creation of the links (either symbolic or hard) in two cases: the source file does
```

Comment:
It should be rephrased that this is only possible for the second case: `the destination exists and is a file`.

Reply:
no, the first case only works for symlink, but the 2nd case works for both